db/schema.rb

```diff
@@ -0,0 +1,22 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_220_719_112_715) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'greetings', force: :cascade do |t|
+    t.string 'greeting'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+  end
```

Comment:
- [ ] Well done creating the greeting table. Kindly create 5 greetings in the seed file. Otherwise, when users open the react app they should create greetings by themselves on the side of the rail. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md#:~:text=Create%20a%20table%20for%20storing%20your%20messages%20and%20create%205%20different%20greetings.).